release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2011,6 +2020,11 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|Intel E810-XXVDA4T
```

Comment:
It does seem like not all the NICs that we have supported are in this table. E.g.
```
|Broadcom
|BCM57414
|14e4
|16d7
|Broadcom
|BCM57508
|14e4
|1750
|Intel
|X710
|8086
|1572
|Intel
|XL710
|8086
|1583
|Intel
|XXV710
|8086
|158b
|Intel
|E810-CQDA2
|8086
|1592
|Intel
|E810-2CQDA2
|8086
|1592
|Intel
|E810-XXVDA2
|8086
|159b
|Intel
|E810-XXVDA4
|8086
|1593
```

Reply:
@wizhaoredhat That is likely. We don't add every single device to this table. As a Technology Preview table, we keep entries in for 3 releases (for example, for 4.13 the table is for 4.11, 4.12, and 4.13). Unless the devices you listed above were added in one of those releases, that specific table will not list them. 

We provide the full list of devices here: https://docs.openshift.com/container-platform/4.12/networking/hardware_networks/about-sriov.html#supported-devices_about-sriov 

**If any of these devices were added in 4.13, 4.12, or 4.11, and we missed them, I can add them to this list.**

Reply:
Ah I see. Thanks for clarifying!